03_it_analysis/src/it_analysis_data_prep.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Nov 29 10:17:00 2021
+
+@author: ggorski
+"""
+
+import boto3
+import pandas as pd
+import io
+import subprocess
+import os
+
+#%%
+def download_s3_to_local(s3_dir_prefix, local_outdir, file_id):
+    '''download data files from s3 bucket to local machine for development
+    file_id - a file identifier substring that is contained within all 
+    the file names you want to download. For example 'usgs_nwis' will 
+    download all files with 'usgs_nwis' in the file name'''
+    
+    # after you have configured saml2aws, you can log in and create a new 
+    # token by executing the following command:
+    # keep all of the s3 arguments hard coded as I don't see us changing them much
+    subprocess.run(["saml2aws", "login", "--skip-prompt", "--role", "arn:aws:iam::807615458658:role/adfs-wma-developer"])
```

Comment:
I should stick this in the util function. that I add below..I will update it so it doesn't need to be run here
```suggestion
```